diktat-rules/src/test/kotlin/org/cqfn/diktat/ruleset/chapter4/AccurateCalculationsWarnTest.kt

```diff
@@ -150,4 +150,32 @@ class AccurateCalculationsWarnTest : LintTestBase(::AccurateCalculationsRule) {
                 LintError(16, 9, ruleId, warnText("x", "x %= 2"), false)
         )
     }
+
+    @Test
+    @Tag(WarningNames.FLOAT_IN_ACCURATE_CALCULATIONS)
+    fun `should allow arithmetic operations inside abs in comparison`() {
+        lintMethod(
+                """
+                    |import kotlin.math.abs
+                    |
```

Comment:
I would also add test with `==`

Reply:
As far as I understand, `==` should not be allowed in this case and it is checked in rule 4.1.1
Or do you mean that we need a test that it is still not allowed?

Reply:
@akuleshov7 I've added test with `==`